concordium-std/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased changes
 
 - Add a new primitive `get_random` for generating random numbers in Wasm code testing; `get_random` can be used in tests only, not available for smart contracts on the chain.
+- Move the type `AccountBalance` to `concordium-contracts-common`.
```

Comment:
I think you are well-behind in this branch. 6.0.0 and 6.0.1 have both been released already.

Reply:
Seems like I am, yes :p I'll fix that.